src/Uniqueness.php

```diff
@@ -4,6 +4,14 @@
 
 trait Uniqueness
 {
+    /**
+     * start - Start the uniqueness command with the given actionID.
```

Comment:
You shouldn't repeat the method name itself.  `Start the uniqueness command with the given actionID.` would be a good choice. It should be done in other doc-blocks too.